apex_legends/__init__.py

```diff
@@ -1,5 +1,6 @@
-from .base import ApexLegends
+from .base import ApexLegends, AsyncLegends
```

Comment:
Maybe AsyncApexLegends would be better

Reply:
I think the opposite.
AsyncLegends is such a good play-on to the original class, and clearly tells you its purpose without adding extra words into the mix.